docs/requirements.txt

```diff
@@ -7,3 +7,5 @@ mkdocstrings>=0.28.2
 mkdocstrings-python>=1.16.2
 mkdocs-gen-files==0.5.0
 mkdocs-literate-nav==0.6.1
+# mkdocstrings-python will format the docstrings using ruff (https://mkdocstrings.github.io/python/usage/configuration/signatures/#line_length)
+ruff>=0.9.6
```

Comment:
Nit, I'd rather pin this (and other dependencies, tbh) like we do in all our other `requirements.txt` files.

Reply:
Resolved in 0b0a9fc